code/game/machinery/cryo.dm

```diff
@@ -399,7 +399,7 @@
 	expel_gas = air_contents.remove(remove_amount)
 	expel_gas.temperature = T20C // Lets expel hot gas and see if that helps people not die as they are removed
 	loc.assume_air(expel_gas)
-	air_update_turf(1)
+	air_update_turf()

```

Comment:
1 is used to force an adjacent turfs check, which something like cryo, when expelling gas, kinda needs to know, the whole "where can I spread this" to.